src/styles/index.scss

```diff
@@ -1,3 +1,113 @@
+@import '../utils/variables';
+@import './reset.css';
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 200px;
```

Comment:
The max-width property limits the maximum width of an element, which may lead to unexpected behavior when the parent container is smaller than the max-width. Consider using the width property instead.